buildSrc/src/main/groovy/GenerateDataFromManPages.groovy

```diff
@@ -230,8 +230,10 @@ class GenerateDataFromManPages extends DefaultTask {
           for (String section : sections) {
             logger.debug("Found options $section in $option in ${file.getAbsolutePath()}")
             sectionToKeyWordMapFromDoc.putIfAbsent(section, new TreeMap<>())
-            sectionToKeyWordMapFromDoc[section][keyName] =
-              ["values": keyValue, "declaredUnderKeyword": originalKeyName, "declaredInFile": filename]
+            def val = ["declaredInFile": filename]
```

Comment:
Is there a reason to change this? Is it just to improve memory?

Reply:
To reduce json size, not serializing 'default' values.